models/topics.models.js

```diff
@@ -1,7 +1,16 @@
 const db = require("../db/connection.js");
+const { readFile } = require("fs/promises");
 
 exports.fetchAllTopics = () => {
   return db.query("SELECT * FROM topics").then((result) => {
     return result.rows;
   });
 };
+
+exports.fetchAllEndpoints = () => {
+  return readFile("./endpoints.json", "utf-8").then((file) => {
```

Comment:
It's fine to create a model that deals with reading the file but isn't necessary, as there is no interaction with the database for this endpoint, it can be done in the controller.